scripts/versioningCheck.sh

```diff
@@ -6,15 +6,34 @@ CURRENT_BRANCH=`git branch --show-current`
 
 git checkout $LATEST_RELEASE
 
+
+if (command -v nvm &> /dev/null)
+then
+  NODE_MANAGER="nvm"
+elif (command -v n &> /dev/null)
+then
+  NODE_MANAGER="n"
+elif (command -v fnm &> /dev/null)
+then
+  NODE_MANAGER="fnm"
+else
+  echo "No node manager found"
+  exit 1
+fi
+
 # Compile release
-npm ci --force && npx truffle compile
-rm -rf build-$LATEST_RELEASE || true
-mv build build-$LATEST_RELEASE
+$NODE_MANAGER install
```

Comment:
Do all node managers have the same interface?

Reply:
I have tested it with `nvm` (on Circle) and `fnm` (locally). I haven't tested with `n`, but the docs seemed to suggest that was the case.